pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -1524,12 +1519,10 @@ private void recreateStreamConsumer(String reason) {
   /**
    * Creates a new stream metadata provider
    */
-  private void makeStreamMetadataProvider(String reason) {
-    if (_streamMetadataProvider != null) {
-      closeStreamMetadataProvider();
-    }
-    _segmentLogger.info("Creating new stream metadata provider, reason: {}", reason);
-    _streamMetadataProvider = _streamConsumerFactory.createStreamMetadataProvider(_clientId);
+  private void createPartitionMetadataProvider(String reason) {
+    closeStreamMetadataProvider();
```

Comment:
nit: s/_streamMetadataProvider/_partitionMetadataProvider and s/closeStreamMetadataProvider/closePartitionMetadataProvider ?

Reply:
done